ui/apps/dashboard/src/components/Navigation/OnboardingWidget.tsx

```diff
@@ -0,0 +1,98 @@
+import Link from 'next/link';
+import { NewButton } from '@inngest/components/Button';
+import { MenuItem } from '@inngest/components/Menu/MenuItem';
+import SegmentedProgressBar from '@inngest/components/ProgressBar/SegmentedProgressBar';
+import { Tooltip, TooltipContent, TooltipTrigger } from '@inngest/components/Tooltip/Tooltip';
+import { RiBookReadLine, RiCheckboxCircleFill, RiCloseLine } from '@remixicon/react';
+import { useLocalStorage } from 'react-use';
+
+import { useBooleanFlag } from '@/components/FeatureFlags/hooks';
+import { pathCreator } from '@/utils/urls';
+import { onboardingWidgetContent } from '../Onboarding/content';
+import { type OnboardingSteps } from '../Onboarding/types';
+
+type OnboardingStepsString = `${OnboardingSteps}`;
+
+export default function OnboardingWidget({
+  collapsed,
+  closeWidget,
+}: {
+  collapsed: boolean;
+  closeWidget: () => void;
+}) {
+  const { value: onboardingFlow } = useBooleanFlag('onboarding-flow-cloud');
+
+  const [onboardingWidgetStep] = useLocalStorage<OnboardingStepsString>(
+    'onboardingWidgetStep',
+    '1',
+    { raw: true }
+  );
+  const currentStep: OnboardingSteps = Number(onboardingWidgetStep) as OnboardingSteps;
+  const finalStep = currentStep === 4;
+  const stepContent = onboardingWidgetStep
+    ? onboardingWidgetContent.step[currentStep]
+    : onboardingWidgetContent.step[1];
+
+  if (!onboardingFlow) return;
+  return (
+    <>
+      {collapsed && (
```

Comment:
A since ternary instead of two separate mutually exclusive checks on collapsed would probably be a bit more readable.

Reply:
Do you usually prefer the ternary? Sometimes I find them harder to read, on bigger JSX items I usually end up splitting them